src/css/augmentedsteam.css

```diff
@@ -1358,10 +1358,6 @@ img.astats_icon {
   overflow-x: initial !important;
   overflow: visible !important;
 }
-.es_regional_prices.game_area_purchase_game:hover,
-.es_regional_prices.sale_page_purchase_item:hover {
-  z-index: 9999;
-}
```

Comment:
I'm not clear what these styles are for (added in https://github.com/jshackles/Enhanced_Steam/commit/e96b382abe9017a2a958d54502ffa7124c5f3d7e) but they glitch out the dialog so I removed them.

Reply:
Turns out this is needed for the regional price popup to show on top of subsequent purchase containers. I think lowering the z-index will fix the dialog glitch.